boards/nxp/frdm_mcxe247/frdm_mcxe247.dts

```diff
@@ -160,6 +160,10 @@
 	clock-div = <1>;
 };
 
+&flexio0 {
+	status = "okay";
```

Comment:
Does it make sense without tests/drivers/pwm/pwm_api/boards/frdm_mcxe247_flexio_pwm.overlay?
Should it be enabled in the overlay?